distributions/standalone/src/main/resources/MIGRATION.txt

```diff
@@ -290,3 +290,4 @@ MULE-7524: Encoded mule expressions in dynamic endpoints address components are
 MULE-7546: Implementations of org.mule.api.MuleMessage now need to implement clearAttachments() which was added to the interface. Implementations that extend org.mule.el.context.AbstractMapContext must now implement clear() given this method has been removed from the abstract implementation.
 MULE-7612: maps returned by queries using the Database connector can have different keys as now column's aliases are used instead of column's names.
 MULE-8066: Since 3.6 MuleClient uses the new HTTP module for processing HTTP URLs. To use the previous behaviour use the attribute 'useTransportForUris' in the http:config  element that's located inside the mule configuration element. You can also set this globally by using the system property 'mule.http.useTransportForUris'.
+MULE-11825: In a db template query, to set a db param with the null value, you can use the "null" literal value. For example: <db:in-param name="name" defaultValue="null"/>
```

Comment:
My previous comment was "I would only accept NULL to represent a null.", any reason you choose "null" instead?